db/models.py

```diff
@@ -0,0 +1,27 @@
+from sqlalchemy import Column, Integer, String, ForeignKey, Date
+from sqlalchemy.orm import relationship
+
+from db.engine import Base
+
+
+class DBAuthor(Base):
+    __tablename__ = "author"
+
+    id = Column(Integer, primary_key=True)
+    name = Column(String(255), nullable=False, unique=True)
+    bio = Column(String(255), nullable=False)
+    books = relationship("DBBook", back_populates="author")
+
+
+class DBBook(Base):
+    __tablename__ = "book"
+
+    id = Column(Integer, primary_key=True, index=True)
+    title = Column(String(255), nullable=False)
+    summary = Column(String(255))
+    publication_date = Column(Date, nullable=False)
+    author_id = Column(Integer, ForeignKey("author.id"))
+    author = relationship("DBAuthor", back_populates="books")
+
+    def str(self):
+        return f"{self.publication_date.strftime("%Y-%m-%d")}"
```

Comment:
The `str` method should be defined as `__str__` to properly override the string representation of the object. Additionally, ensure that the `publication_date` is not `None` before calling `strftime` to avoid potential errors.